### ISSUE TITLE
Adding source to location card

### ISSUE THREAD

**Issue by hannaheneil** [postmanlabs/covid-19-apis#217]
Can we add the source URL to the individual location cards? I think that will aid the legitimacy and is good information to check in case it becomes outdated.

**Comment by ch264** [comment]
@hannaheneil can you please clarify what you mean by source url? Do you mean a link to the site on the 'state list page'?

**Comment by hannaheneil** [comment]
No, there is a field for the source URL in the spreadsheets, on the last tab. Kind of like how the transportation field was hidden, could we reveal this field?

**Comment by kinlane** [comment]
Yes I have it on my list to expose via the JSON -- will prioritize adding.
